climpred/classes.py

```diff
@@ -24,14 +24,14 @@
 # PerfectModel:
 # TODO: add relative entropy functionality
 
-# Reference
+# Hindcaste
```

Comment:
Is there supposed to be a suffix `e`?

Reply:
I noticed that 😂 Going to revisit this once the other PR is merged. Just wanted to get the other stuff in so I could rebase and then implement this. (Need to fix a few notebooks here)